docs/data/routes/guides/wip/migration-guide/en.md

```diff
@@ -0,0 +1,49 @@
+---
+name: migration-guide
+routeTemplate: ./data/component-templates/guide.yml
+title: Migration Guide
+---
+
+> This page is still a work in progress. ⚠️ Help Wanted.
+
+## Migrate from MVC to JSS
+Before we get started with converting an existing MVC site to JSS, lets answer a few common questions.
+- Do we need a separate JSS solution?
+No, the MVC implementation and JSS implementation can exist in the same solution, however, the front-end application is typically in a separate folder.
+
+- Can we incrementally refactor each component?
+Yes and no. MVC and JSS structures are fundamentally different, thus it is not possible to mix MVC and JSS renderings. This means you will not be able to replace each MVC rendering with a JSS rendering on an existing page as you migrate them.
+
+- We just need to add Node.js to our CD servers, right?
+Your infrastructure will more than likely need to change depending on your selected [deployment topology](/docs/techniques/devops#choosing-a-deployment-topology).
+
+With these concepts in mind, consider the following approach:
+
+**1. Configure JSS Application**
+Perform the [JSS Server Setup](/docs/getting-started/jss-server-install) and [App Deployment](/docs/getting-started/app-deployment) followed by creating the configuration file for your application. The configuration file varies based on your selected JavaScript Framework. Sitecore provides [example configurations](https://github.com/Sitecore/jss/tree/master/samples)], such as the [JssReactWeb.config](https://github.com/Sitecore/jss/blob/master/samples/react/sitecore/config/JssReactWeb.config).
```

Comment:
It seems that this section of markdown renders the extra ']' symbol on line 23 "Sitecore provides [example configurations]"